tokio-console/src/view/mod.rs

```diff
@@ -144,6 +144,12 @@ impl View {
                         self.state = ResourcesList;
                         update_kind = UpdateKind::Other;
                     }
+                    key!(Char('r')) => {
+                        self.state = ResourcesList;
+                    }
+                    key!(Char('t')) => {
+                        self.state = TasksList;
+                    }
```

Comment:
take it or leave it: it occurs to me that we _could_ potentially reduce duplication a bit by pulling this code out to _before_ the match here: https://github.com/tokio-rs/console/blob/5621004bf5fa620f945f559daea3c9aa2d4cedab/tokio-console/src/view/mod.rs#L99 so that we have an `if` that handles the `r` and `t` keys regardless of what view we're in, rather than duplicating it for each potential view.

currently, these keys are handled inside the match because the Tasks view doesn't switch back to itself when the `t` key is pressed, and the Resources view similarly skips the `r` key. however, this probably isn't necessary --- changing `self.state` won't result in redrawing the view at a different position or anything.

so, we could probably simplify this code a bit by handling the `r` and `t` keys outside of the `match`, if you think that would be nicer? not a blocker though.

Reply:
Ah, do you mean, if `r` or `t`; do the right thing; else do the match on line 99?